share/smack/svcomp/utils.py

```diff
@@ -16,19 +16,21 @@ def svcomp_frontend(args):
   if len(args.input_files) > 1:
     raise RuntimeError("Expected a single SVCOMP input file.")
 
-  # test float\bv benchmarks
+  # test bv and executable benchmarks
   file_type = filters.svcomp_filter(args.input_files[0])[0]
   if file_type == 'bitvector':
     args.bit_precise = True
-  if file_type == 'float':
-    sys.exit(smack.top.results(args)['unknown'])
   args.execute = False
   if filters.svcomp_filter(args.input_files[0])[1] == 'executable':
     args.execute = True
 
   name, ext = os.path.splitext(os.path.basename(args.input_files[0]))
```

Comment:
@zvonimir I think it can. What about not invoking `svcomp_filter` if memory safety benchmark is detected?